be/src/common/config.cpp

```diff
@@ -1169,7 +1169,9 @@ DEFINE_mInt64(file_cache_background_lru_dump_update_cnt_threshold, "1000");
 DEFINE_mInt64(file_cache_background_lru_dump_tail_record_num, "5000000");
 DEFINE_mInt64(file_cache_background_lru_log_replay_interval_ms, "1000");
 DEFINE_mBool(enable_evaluate_shadow_queue_diff, "false");
-
+DEFINE_mBool(enable_file_cache_normal_queue_2qlru, "true");
```

Comment:
default value better to be false as it is not widely used.

is there any test results to show how much this mechanism improves the hit rate? 

is there any upgrade/downgrade compatibility issues if a new queue `cold_normal` and new file name `_cold_normal` introduced?